source/agora/node/Ledger.d

```diff
@@ -667,15 +689,17 @@ version (unittest)
                 ? params_
                 : (blocks.length > 0
                    // Use the provided Genesis block
-                   ? new immutable(ConsensusParams)(cast(immutable)blocks[0], WK.Keys.CommonsBudget.address)
+                   ? new immutable(ConsensusParams)(cast(immutable)blocks[0], WK.Keys.CommonsBudget.address,
+                            1008, 7, 80, 30, 1, 1024, 200, 600)
```

Comment:
Why not keep these in the default parameters list in the ctor? That's why they were made, so they don't have to be copy-pasted to the call site.

Reply:
the last parameter is NOT the default, that is why it has to be repeated

Reply:
> the last parameter is NOT the default, that is why it has to be repeated

Are you sure? That's not what I see. Unless you pushed an update that is.

Reply:
I am 99% sure :) the test uses 600 secs tolerance while the real one uses 60 secs. I did one push, but that was to remove an unnecessary commit, not related to this.

Reply:
> the test uses 600 secs tolerance while the real one uses 60 secs

I mean, why not set it to 600 here then https://github.com/bpfkorea/agora/blob/da2c738e94c8918ec2356154f11737ae22b9fe88/source/agora/consensus/data/Params.d#L108 ?

Reply:
there are 2 versions of the constructor for ConsensusParameter
1. accept a block as the genesis block in the first parameter (this is the version that is used by prod code and by the test you referenced)
2. doesn't accept any blocks

I don't think setting the consensus block_timestamp_tolerance_secs for the 1st type of constructor to 600 is a good idea, as that doesn't seem to be a reasonable tolerance for the prod code.